cypress/e2e/open.spec.js

```diff
@@ -51,12 +51,13 @@ describe('Open existing office files', function() {
 			cy.waitForViewer()
 			cy.waitForCollabora()
 
-			cy.screenshot('open-file_' + filename)
+			cy.waitForPostMessage('App_LoadingStatus', { Status: 'Document_Loaded' })
 
 			// Share action
+			cy.wait(2000)
```

Comment:
It looks like there is some UI reset once the document renders, but I cannot figure out a proper way to assert that it is actually ready. Would go with the wait for now to move on.

Cypress run recording for reference:

https://github.com/nextcloud/richdocuments/assets/3404133/41dd2339-4863-4d5f-bb42-a01b38c2eeca

FYI @pedropintosilva Nothing critical i think, though wondering why the UI is reset there